pkg/metrics/metrics.scss

```diff
@@ -284,23 +289,41 @@
         grid-row: 0;
         grid-column: 1 / -1;
         background: #eee;
+        margin-left: var(--pf-global--spacer--md);
     }
 
     &-events {
         grid-column: events;
         grid-row: calc(var(--metrics-minutes) - var(--metrics-minute));
-        display: grid;
-        grid-template-columns: [info] 1fr [time] 1fr;
-        align-items: baseline;
 
-        dt {
-            grid-column: time;
-            font-size: var(--pf-global--FontSize--sm);
-            grid-row: 1;
+        .description {
+            display: grid;
+            grid-template-columns: [info] 1fr [time] 1fr;
+            align-items: baseline;
+            justify-items: start;
+
+            .time {
+                grid-column: time;
+                font-size: var(--pf-global--FontSize--sm);
+                grid-row: 1;
+                display: flex;
+                justify-content: space-between;
```

Comment:
This is kinda working as expected but it's a confusing approach.
.time class wraps the time and the all-logs link. In this case we should either name it differently or just let the outer grid handle the spacing. And I prefer the latter.

So we can completely remove this class if we have the outer grid be live:

grid-template-columns: [info] 1fr [time] 1fr [link];
And let's add some spacing with grid-column-gap. Then we can mostly drop the .time class content.

Reply:
Actually there is a reason why I did this this way. When you toggle open one item, the time gets mis-aligned with other items:

Currently:
![alignedtimes](https://user-images.githubusercontent.com/12330670/110844943-50685c80-82aa-11eb-9589-76ac9be754d1.png)

With using ` [info] 1fr [time] 1fr [link];` (notice how times are not aligned anymore)
![withgrid](https://user-images.githubusercontent.com/12330670/110844977-5b22f180-82aa-11eb-9249-a30351d529e6.png)

I'll rename it to make it less confusing.